aspnet: note on proxy ip (#4892)
getsentry/sentry-docs: src/platforms/dotnet/common/troubleshooting.mdx

description: "Learn more about how to troubleshoot common issues with the .NET SDK. "
---

## User IP address in Sentry shows up as the server IP

If you're using a proxy server that relies on `X-Forwarded-For`, you might need to [configure ASP.NET Core

> so that it's aware of it](https://docs.microsoft.com/aspnet/core/host-and-deploy/proxy-load-balancer).

Reviewer: "so that it's aware of it" is a bit awkward since the two its aren't the same thing. Suggest: 

```suggestion
so that it's aware of the request header](https://docs.microsoft.com/aspnet/core/host-and-deploy/proxy-load-balancer).
```

> See this [GitHub issue for more context](https://github.com/getsentry/sentry-dotnet/issues/258)

Reviewer: Suggest this placement of the link text

```suggestion
See [this GitHub issue] (https://github.com/getsentry/sentry-dotnet/issues/258) for more context
```

> See [this GitHub issue] (https://github.com/getsentry/sentry-dotnet/issues/258) for more context

Reviewer: just realized this is missing a period.

```suggestion
See [this GitHub issue] (https://github.com/getsentry/sentry-dotnet/issues/258) for more context.
```
